enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -246,6 +243,27 @@ public void activateUser( @Name( "username" ) String username ) throws IOExcepti
 
         subject.getUserManager().getUser( username );
 
+        return terminateSessionsForValidUser( username );
+    }
+
+    // ----------------- helpers ---------------------
+
+    private Stream<TransactionTerminationResult> terminateTransactionsForValidUser( String username )
+    {
+        Long killCount = 0L;
+        for ( KernelTransaction tx : getActiveTransactions() )
+        {
+            if ( tx.mode().name().equals( username ) && tx != this.tx )
```

Comment:
So should we fix that this will miss transactions that are currently executing procedure calls on this PR or deal with it later? (Previous comments got hidden by changing diffs)